lizzy/version.py

```diff
@@ -1,2 +1,2 @@
 # Version to be checked on the client
-VERSION = '2016-02-22'
+VERSION = '1.4.2'
```

Comment:
So that is the justification of having two versions. So when do we update the `VERSION` in the date based version? Whenever the API change?